src/analysis/mean_inequalities.lean

```diff
@@ -802,6 +802,53 @@ begin
   end
 end
 
+lemma lintegral_Lp_mul_le_Lq_mul_Lr {α} [measurable_space α] {p q r : ℝ} (hp1 : 1 ≤ p)
```

Comment:
I guess you just need `0 < p`, here and in the applications.

Reply:
Indeed